### PR TITLE
Rework the entire config

### PR DESCRIPTION
**Pull request by NextWork123**
Hello, i'm opening this pull request for fix much issue on the config and implement a script for cachyos-rate-mirrors. I suggest to test that changes,if someone found some error contact me.

### CODE REVIEW

File: etc/skel/.config/hypr/hyprland.conf [hunk 1]
```diff
@@ -116,7 +117,7 @@ bind=SUPER,space,exec,wofi --show drun -o DP-3
 bind=SUPER,P,pseudo,
 bind=SUPER,C,exec, kitty --class neovim -e nvim
 bind=SUPER_SHIFT,T,exec, todoist
```
Comment (vnepogodin) [inline etc/skel/.config/hypr/hyprland.conf:119]:
```ini
bind=SUPER_SHIFT,T,exec, todoist
```

This should be removed or commented

File: etc/skel/.config/hypr/hyprland.conf [hunk 2]
```diff
@@ -164,3 +165,8 @@ bind=CTRL,1,exec,kitty --title fly_is_kitty --hold cava
 bind=CTRL,2,exec,code-insiders
```
Comment (vnepogodin) [inline etc/skel/.config/hypr/hyprland.conf:165]:
```ini
bind=CTRL,2,exec,code-insiders
```

This should be also removed or commented

File: etc/skel/.config/hypr/hyprland.conf [hunk 3]
```diff
@@ -164,3 +165,8 @@ bind=CTRL,1,exec,kitty --title fly_is_kitty --hold cava
...
 bind=CTRL,3,exec,kitty --single-instance --hold donut.c
```
Comment (vnepogodin) [inline etc/skel/.config/hypr/hyprland.conf:166]:
```ini
bind=CTRL,3,exec,kitty --single-instance --hold donut.c
```

This should be also removed or commented

File: etc/skel/.config/hypr/hyprland.conf [hunk 4]
```diff
@@ -164,3 +165,8 @@ bind=CTRL,1,exec,kitty --title fly_is_kitty --hold cava
...
 bind=CTRL,4,exec,kitty --title clock_is_kitty --hold tty-clock -C5
```
Comment (vnepogodin) [inline etc/skel/.config/hypr/hyprland.conf:167]:
```ini
bind=CTRL,4,exec,kitty --title clock_is_kitty --hold tty-clock -C5
```

This should be also removed or commented

**Review by vnepogodin — CHANGES_REQUESTED** [review]

**Comment by NextWork123** [comment]
Removed all this bind need only a testing by Sewdohe

**Review by ptr1337 — APPROVED** [review]

**Review by vnepogodin — APPROVED** [review]

**Pull request merged**